frontend/src/pages/Signup.jsx

```diff
@@ -0,0 +1,116 @@
+import React, { useState, useEffect } from 'react';
+import '../components/Styles/Signup.css'
+import axios from "axios";
+import { createRoot } from "react-dom/client";
+
+
+const Signup = () => {
+
+//   const [userid, setUserid] = useState("");
+//   const [firstname, setFirstname] = useState("");
+//   const [lastname, setLastname] = useState("");
+//   const [username, setUsername] = useState("");
+//   const [password, setPassword] = useState("");
+//   const [birthday, setBirthday] = useState("");
+//   const [age, setAge] = useState("");
+//   const [gender, setGender] = useState("");
+
+//   const handleClick = () => {
+//     axios.post('https://localhost:8800/signup',
+//     {
+//         userid: userid,
+//         firstname: firstname,
+//         lastname: lastname,
+//         username: username,
+//         password: password,
+//         birthday: birthday,
+//         age: age,
+//         gender: gender
+//     }
+//   )
+//     .then((response) => {
+//         console.log(response);
+//     });
+const [formData, setFormData] = useState({
+    userid: 0,
```

Comment:
in database, userID is autoincremented
no need for user to assing themseselves their id
what if two user will want the same id at the same time?

Reply:
so i should delete this user id in my form ?